Example/InstanceID/Tests/FIRInstanceIDTest.m

```diff
@@ -157,7 +158,7 @@ - (void)testTokenShouldBeRefreshedIfCacheTokenNeedsToBeRefreshed {
                        handler:[OCMArg any]];
 
   [self.mockInstanceID didCompleteConfigure];
-  OCMVerify([self.mockInstanceID fetchDefaultToken]);
+  OCMVerify([self.mockInstanceID defaultTokenWithHandler:[OCMArg any]]);
```

Comment:
We may verify that handler is `nil` here. I guess, `OCMVerify([self.mockInstanceID defaultTokenWithHandler:nil]);` should work.